csrc/optimization/alias_analysis.h

```diff
@@ -61,14 +61,14 @@ class AliasAnalysisResult {
 
   std::string toString(int indent_size) const;
 
-  // Gets the aliased fusion input of a fusion output. Returns nullptr
-  // when `fusion_out` is not a fusion output or does not alias a fusion input.
-  TensorView* getAliasedInput(const TensorView* fusion_out) const;
+  // Gets the nearest aliased fusion input of a `fusion_out` other than
```

Comment:
```suggestion
  // Gets the nearest aliased fusion input/output of a `fusion_out` other than
```
We can also have output -> output aliases now right?